### ISSUE TITLE
probability distribution from AMPLIFY

### ISSUE THREAD

**Issue by ShuhaoGithub** [chandar-lab/AMPLIFY#9]
Dear authors,

Thank you for this amazing tool! I am eager to try it and compare with ESM. I was wondering if it is possible to obtain the amino acid probability matrix (20*L) from the embedding? I found the code to get the embedding but not the probs. Thanks!

**Comment by qfournier** [comment]
Thank you for your interest! 

The output of the forward pass is `MaskedLMOutput(logits, hidden_states, attentions)`. You can retrieve the logits and then pass them through a `Softmax` to compute the probabilities.

```python
input = tokenizer.encode(sample, return_tensors="pt").to("cuda")
output = model(input)
logits = output["logits"]
probabilities = torch.nn.functional.softmax(logits, dim=-1)
```

**Comment by ShuhaoGithub** [comment]
Wonderful! Thank you!
